### ISSUE TITLE
Make ps_imageslider compliant with v9 .htaccess policy for direct PHP calls

### ISSUE THREAD

**Issue by matks** [PrestaShop/PrestaShop#36146]
Update module ps_imageslider to make it compliant with https://github.com/PrestaShop/PrestaShop/pull/34184
Then perform a module release

Once this is done, the issue https://github.com/PrestaShop/PrestaShop/pull/34184 can be closed.

The work started in PR https://github.com/PrestaShop/ps_imageslider/pull/96 could help

**Comment by Hlavtox** [comment]
This is already tracked here - https://github.com/PrestaShop/PrestaShop/issues/34578

**Comment by matks** [comment]
Fixed by https://github.com/PrestaShop/ps_imageslider/pull/96
